scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+        
+    except Exception as e:
+        if 'temp_file' in locals():
+            temp_file.close()
+            import os
+            if 'temp_path' in locals() and os.path.exists(temp_path):
+                os.unlink(temp_path)
+        gc.collect()
+        return pref_zip_name, None, 0
+
+
+def process_archive_parallel(archive_path: Path, num_workers: int = None):
```

Ruff (0.12.2)

`process_archive_parallel` is too complex (16 > 10)

(C901)